examples/asr/transcribe_speech.py

```diff
@@ -212,6 +231,11 @@ def autocast():
         return cfg
 
     # transcribe audio
+
+    return_hypotheses = False
```

Comment:
Make return hypothesis set to true by default. High time we moved to the more general interface. Can you update the rest of the script to use Hypothesis outputs ? It's not technically part of LID but might as well do the cleanup in this pr to group related changes.

Reply:
done